sep dev update (#111)
irods/irods.org: content/posts/irods-development-update-september-2025.md

Title: iRODS Development Update: September 2025
Date: 2025-09-17 10:00
Author: Kory Draughn
Slug: irods-development-update-september-2025
Status: published

Hello reader,

> For this month's development update, we've started brainstorming how to improve data transfers. This has led us to reviewing the rsync delta-transfer algorithm and how that could fit into iRODS. We've also started looking into making the server FIPS compliant. As a first step, we're looking into moving away from MD5 within the server and iCommands in a backward compatible way. This will make iRODS available to organizations which require FIPS in their environment.

Reviewer: ```suggestion
For this month's development update, we've started brainstorming how to improve data transfers. This has led us to reviewing the rsync delta-transfer algorithm and how that could fit into iRODS. We've also started looking into making the server FIPS compliant. This means removing all uses of MD5 within the server and iCommands in a backward compatible way. This will make iRODS available to organizations which require FIPS in their environment.
```
